resources/views/layout/admin/app.blade.php

```diff
@@ -0,0 +1,4 @@
+@include('layout.admin.header')
+@include('layout.admin.side-bar')
+@yield('content')
+@include('layout.admin.footer')
```

Comment:
Theo mình file app.blade.php sẽ chứa đường dẫn của css và script. Dùng @yield như sau : 
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <!-- Tell the browser to be responsive to screen width -->
  <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
  <link href="{{ URL::asset("bower_components/bootstrap/dist/css/bootstrap.min.css") }}" rel="stylesheet">
 ......
 @yield('title_page')
</head>
<body class="hold-transition skin-blue sidebar-mini">
  @include('layouts.admin.header')
  @include('layouts.admin.menu')

  <div class="wrapper">
  @yield('content')
  </div>
  @include('layouts.admin.footer')
  <script src="{{ URL::asset("bower_components/jquery/dist/jquery.min.js") }}"></script>
 ....
  @yield('script')
</body>
</html>